src/style-guide/README.md

```diff
@@ -210,19 +210,9 @@ When Vue processes directives, `v-for` has a higher priority than `v-if`, so tha
 </ul>
 ```
 
-Will be evaluated similar to:
+Will throw an error, because `v-if` directive will be evaluated first and property `user` does not exist at this moment.
```

Comment:
I'd put a 'the' before `v-if`.

I'd also switch 'property' to 'the variable' or 'the iteration variable' or something like that.

Reply:
Thank you, makes sense to me, fixed 🎉